src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/RazorFoldingRangeRequestParam.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+using Microsoft.AspNetCore.Razor.LanguageServer.Folding;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts
+{
+    internal class RazorFoldingRangeRequestParam : VSFoldingRangeParamsBridge
```

Comment:
We plan to get rid of the bridge types, I'd have this inherit from `FoldingRangeParams` and move this request type over to the LanguageSErver.Protocol project so it can be easily shared without requiring a reference to the language server project